incident-response-plan.md

```diff
@@ -0,0 +1,153 @@
+# Incident Response Plan
+
+## Purpose  
+This document outlines the process for handling security-impacting incidents affecting one or more projects in the Foundation’s ecosystem. Incidents may include platform changes with security implications, account compromise, or other security events that require coordinated response.
+
+The Foundation’s role is to:  
+1. **Receive and triage THE REPORTS**  
+2. **Connect reporters and affected maintainers with the right experts**  
+3. **Facilitate coordinated response** across multiple projects when needed  
+4. **Communicate clearly and act as the contact point** while respecting confidentiality and responsible disclosure principles  
```

Comment:
I think we can nix the "communicate clearly" portion, that is a given. 

**Act as the contact point** while respecting confidentiality and responsible disclosure principles